README.md

```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
 
 **Note**: The recommended restart policy for the container is `on-failed` or `no`
 
+#### Docker Compose Example
+```yaml
+...
+  auto-southwest:
+    image: jdholtz/auto-southwest-check-in
+    container_name: auto-southwest
+    networks:
+      - default
+    security_opt:
+      - no-new-privileges:true
+    restart: on-failed
```

Comment:
It actually should be `on-failure`. That's a mistake in the Readme, so it would be great if you could fix it there as well.

Reply:
fixed in both, thanks: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/3b9fc70f7496d7221ab85ee247c33915c5ecb5c6